omnigibson/prims/cloth_prim.py

```diff
@@ -275,6 +275,17 @@ def particle_velocities(self, vel):
         # the velocities attribute is w.r.t the world frame already
         self.set_attribute(attr="velocities", val=Vt.Vec3fArray.FromNumpy(vel))
 
+    @property
+    def particle_normals(self):
+        """
+        Grabs individual particle positions for this cloth prim
+
+        Returns:
+            np.array: (N, 3) numpy array, where each of the N particles' positions are expressed in (x,y,z)
+                cartesian coordinates relative to the world frame
+        """
+        return np.array(self.get_attribute(attr="normals"))
```

Comment:
Are you sure this works? I think last time when Yunzhu and I tested this, this normals attributes do NOT change when I drag/move the cloth. The values always stay the same.

Reply:
Not being used by other code currently.

Reply:
Did you check if this attribute works?

Reply:
done!